.prettierrc.yml

```diff
@@ -0,0 +1,9 @@
+trailingComma: all
+printWidth: 120
+singleQuote: true
+arrowParens: avoid
+tabWidth: 4
+overrides:
```

Comment:
Need to double check that this is not breaking anything for `static`. I'd love that the tool supported overriding configs from the current folder as ESLint does

Reply:
The site runs fine locally as far as I can tell

Reply:
prettier does search up recursively from the directory of the file being formatted - although I don't think it supports multiple config files.  I believe it stops after finding any config file.